### PR TITLE
Add flink-connector-logger

### PR DESCRIPTION
**Pull request by xinwen-cs**

### CODE REVIEW

**Comment by xinwen-cs** [comment]
@kylemeow

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+*.class
+*.iml
+*.swp
+*.jar
+*.zip
+*.log
+*.pyc
+.DS_Store
+build-target
+flink-runtime-web/web-dashboard/node/
```
Comment (kylemeow) [inline .gitignore:25]: Irrelevant gitignore contents should be removed

File: flink-connector-logger/src/main/java/com/tencent/cloud/oceanus/sink/LoggerTableSinkFactory.java [hunk 2]
```diff
@@ -0,0 +1,342 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
```
Comment (kylemeow) [inline flink-connector-logger/src/main/java/com/tencent/cloud/oceanus/sink/LoggerTableSinkFactory.java:2]: Remove ASF license headers

File: flink-connector-logger/src/main/resources/META-INF/services/org.apache.flink.table.factories.Factory [hunk 3]
```diff
@@ -0,0 +1,16 @@
+# Licensed to the Apache Software Foundation (ASF) under one or more
```
Comment (kylemeow) [inline flink-connector-logger/src/main/resources/META-INF/services/org.apache.flink.table.factories.Factory:1]: Remove ASF header

**Review by kylemeow — CHANGES_REQUESTED** [review]
Thanks for the contribution, and I have made some suggestions on this pull request.

**Review by kylemeow — APPROVED** [review]
OK

**Pull request merged**